.github/workflows/build-and-deploy-on-pypi.yml

```diff
@@ -0,0 +1,60 @@
+name: PyPi Build and Deploy 🐍📦
+
+on:
+  release:
+    types: [published]
+  # use this for testing
+  push:
+    branches:
+      - main
+
+jobs:
+  build-n-publish:
+    name: Build and publish MyProxyClient on PyPi
+    runs-on: ubuntu-latest
+    # may also need this for publishing?
+    environment:
+      name: pypi
+      url: https://pypi.org/project/MyProxyClient/
+    # use the new Trusted Publishers feature
+    # details at https://blog.pypi.org/posts/2023-04-20-introducing-trusted-publishers/
+    # to add a PyPI project to Trusted Publishers, see
+    # https://docs.pypi.org/trusted-publishers/adding-a-publisher/
+    permissions:
+      # IMPORTANT: this permission is mandatory for trusted publishing
+      id-token: write
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.12"
+      - name: Install pep517
+        run: >-
+          python -m
+          pip install
+          pep517
+          --user
+      - name: Build a binary wheel and a source tarball
+        run: >-
+          python -m
+          pep517.build
+          --source
+          --binary
+          --out-dir dist/
+          .
+      #- name: Publish distribution 📦 to Test PyPI
+      #  uses: pypa/gh-action-pypi-publish@master
+      #  with:
+      #    password: ${{ secrets.test_pypi_password }}
+      #    repository_url: https://test.pypi.org/legacy/
+      - name: Publish distribution 📦 to PyPI
+        if: startsWith(github.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@release/v1
+      # this is the old token and password way;
```

Comment:
Could you clean up by removing all the commented out stuff?

Reply:
done!